torch/onnx/utils.py

```diff
@@ -488,7 +488,8 @@ def _export(model, args, f, export_params=True, verbose=False, training=None,
             opset_version=None, _retain_param_name=False, do_constant_folding=True,
             strip_doc_string=True, dynamic_axes=None, keep_initializers_as_inputs=None,
             fixed_batch_size=False, custom_opsets=None, add_node_names=True,
-            enable_onnx_checker=True, use_external_data_format=False):
+            enable_onnx_checker=True, use_external_data_format=False,
+            onnx_shape_inference=False):
```

Comment:
This defaults to False here, but when we we have this baked, we should make this True. Also, hopefully we won't have to have this argument in the public API.